subsys/net/lib/mqtt/mqtt_transport_websocket.c

```diff
@@ -174,8 +174,7 @@ int mqtt_client_websocket_disconnect(struct mqtt_client *client)
 
 	ret = websocket_disconnect(client->transport.websocket.sock);
 	if (ret < 0) {
-		NET_ERR("Websocket disconnect failed (%d)", ret);
-		return ret;
+		NET_DBG("Websocket disconnect failed (%d)", ret);
```

Comment:
This can change the return value, which can now become `0`. Is this an issue?

Reply:
Not really, the only consequence is that we won't get redundant log in case `websocket_disconnect()` fails. The ultimate goal of the function is to close the underlying socket so I think it's ok to return 0 if that succeeds.